client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -110,7 +110,7 @@ class HandleEmailedLinkForm extends React.Component {
 			<EmptyContent
 				action={ action }
 				className={ classNames( {
-					'magic-login__isfetchingauth': isFetching,
+					'magic-login__is-fetching-auth': isFetching,
```

Comment:
I'm pretty sure you can/should drop the 'magic-login__' prefix on this, since it is a modifier.